snakemake/remote/GS.py

```diff
@@ -13,13 +15,42 @@
 try:
     import google.cloud
     from google.cloud import storage
+    from crc32c import crc32
 except ImportError as e:
     raise WorkflowError(
-        "The Python 3 package 'google-cloud-sdk' "
-        "needs to be installed to use GS remote() file functionality. %s" % e.msg
+        "The Python 3 packages 'google-cloud-sdk' and `crc32c` "
+        "need to be installed to use GS remote() file functionality. %s" % e.msg
     )
 
 
+class Crc32cCalculator(object):
```

Comment:
`(object)` is superfluous with Python 3.

Reply:
Fixed.